src/components/ground-nav/ground-nav.scss

```diff
@@ -0,0 +1,262 @@
+@use '../../design-tokens/breakpoint.yml';
+@use '../../design-tokens/colors.yml';
+@use '../../design-tokens/font.yml';
+@use '../../mixins/ms';
+@use '../../mixins/unit';
+
+// Whitespace used for gaps, padding and margin
+// (It uses `rem` so space will be consistent throughout the component)
+$_ground-nav-space: ms.step(1, 1rem);
+// Negative value used to reduce the space around elements as well as position elements.
+$_ground-nav-negative-space: #{unit.strip($_ground-nav-space) * -1}rem;
+$_ground-nav-border-width: 3px;
```

Comment:
Should this correspond to one of our [edge size tokens](https://v-next--cloudfour-patterns.netlify.app/?path=/docs/design-tokens-sizes--page#sizing)?